docker-compose.yml

```diff
@@ -49,6 +49,23 @@ services:
     volumes:
       - .:/app
 
+  # Crontabber app
+  crontabber:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.crontabber
+    image: local/socorro_crontabber
+    env_file:
+      - docker/config/docker_common.env
+      - docker/config/crontabber.env
+    depends_on:
+      - localstack-s3
+      - postgresql
+      - elasticsearch
+      - rabbitmq
```

Comment:
I think that's fair. It definitely needs S3, elasticsearch and (obviously) postgres. I can't think of any crontabber app that connects to rabbit. 

@milescrabill crontabber itself uses the same postgres connection as we have for stuff like productversions, django data, crashes etc. Basically, it uses postgres as the state store but obviously it imports and runs apps (aka. crontabber jobs) that are written in python and need access to things like elasticsearch (e.g. https://github.com/mozilla-services/socorro/blob/master/socorro/cron/jobs/elasticsearch_cleanup.py)